locale/templates/LC_MESSAGES/django.pot

```diff
@@ -4091,6 +4083,27 @@ msgstr ""
 msgid "More information on Developer Accounts"
 msgstr ""
 
+#: src/olympia/devhub/templates/devhub/includes/android_compatibility_modal.html:3
+msgid "Firefox Android compatibility"
```

Comment:
The name of the app is Firefox for Android, not Firefox Android. Same note applies to the following two strings.